Pipelines-and-APIs.md

```diff
@@ -81,7 +81,7 @@ response = requests.post(
 
 Alternatively, binary `files` may be posted instead of plain text `text_files`.
 
-If `text_files` is passed, binary `files` may not be passed and vice-versa. Optional parameters like `output_type` are specified by including them as kwargs in the function signature for `pipeline_api`. They can be passed no matter which parameter is used to send in the file.
+Optional parameters like `output_type` are specified by including them as kwargs in the function signature for `pipeline_api`. They can be passed no matter which parameter is used to send in the file.
```

Comment:
I think this sentence can be deleted actually since response_type / output_format is addressed below.